<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QDialog" name="OptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sheetName</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonNewAverage">
     <property name="text">
      <string>Enter new average grades</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonExistingAverage">
     <property name="text">
      <string>Increase/decrease existing average grades</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
